backend/organization/views/organization_views.py

```diff
@@ -422,6 +478,27 @@ def patch(self, request, url_slug, format=None):
             "organization_size",
             "get_involved",
         ]
+        if "name" in request.data:
+            if Organization.objects.filter(name__iexact=request.data["name"]).exists():
+                return Response(
+                    {
+                        "message": "Organization with name {} already exists".format(
+                            request.data["name"]
+                        )
+                    },
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+            if OrganizationTranslation.objects.filter(
+                name_translation__iexact=request.data["name"]
+            ).exists():
+                return Response(
+                    {
+                        "message": "Organization with name {} already exists".format(
```

Comment:
@JoshKornfeld are we showing this message in the front end? If yes, you should consult with @positiveimpact or @tobiasrehm for a better messaging verbiage. This error message feel more techie. Our user base is not necessarily technical.

Reply:
Changed the error message to something more descriptive.